jiant/proj/simple/preprocessing.py

```diff
@@ -157,13 +158,24 @@ def tokenize_and_featurize(
         List DataRows containing tokenized and featurized examples.
 
     """
-    # TODO: In future, will potentially yield multiple featurized examples
-    #  per example (e.g. SQuAD)
-    # We'll need the 'phase' argument for then
-    data_rows = [
-        example.tokenize(tokenizer).featurize(tokenizer, feat_spec)
-        for example in maybe_tqdm(examples, desc="Tokenizing", verbose=verbose)
-    ]
+    # TODO: Better solution
+    if isinstance(examples[0], squad_style.Example):
+        data_rows = []
+        for example in maybe_tqdm(examples, desc="Tokenizing", verbose=verbose):
+            # TODO more arguments?
```

Comment:
@pyeres thoughts on creating an issue for TODOs and including the issue ID in the comment? I currently don't have context on `more arguments` and likely won't in the future as well

Reply:
I'll add a separate PR after this one is merged to go through and document the TODOs.